app/components/agenda/agendaitem/decision/digital.js

```diff
@@ -361,7 +375,9 @@ export default class AgendaAgendaitemDecisionDigitalComponent extends Component
         newBeslissingHtmlContent = this.intl.t('retracted-item-decision');
         break;
       default:
-        if (this.args.agendaitem.isApproval) {
+        if (subcase?.isBekrachtiging) {
+          newBeslissingHtmlContent = "De Vlaamse Regering bekrachtigt bovengenoemd decreet en beslist over te gaan tot de afkondiging ervan.";
+        } else if (this.args.agendaitem.isApproval) {
```

Comment:
look into `generateApprovalText`, we might want something similar for both decisions and minutes?

Also, scopecreep: Could we use `generateApprovalText` here as well. duplicate code atm

Reply:
"we might want something similar for both decisions and minutes? Since it is only 1 translation without special cases, opted to just use a translation in both scenarios instead of a util.